treehouses/remote
# tunnel: dialog close buttons (fixes #1508, #1521)

Fixes #1508 
Fixes #1521
## How to test:
1. Go to Tunnel
2. Add a port or host
3. The dialogs should now have close buttons.
![Screenshot_20200901-211852_Treehouses Remote](https://user-images.githubusercontent.com/44847682/91932994-b68d7a80-ec9c-11ea-8f69-b6cd96105e87.jpg)
![Screenshot_20200901-211858_Treehouses Remote](https://user-images.githubusercontent.com/44847682/91932995-b7261100-ec9c-11ea-88d3-2b1573f48233.jpg)
![Screenshot_20200901-211905_Treehouses Remote](https://user-images.githubusercontent.com/44847682/91932996-b7bea780-ec9c-11ea-8c42-e103997eccab.jpg)
